clients/android/app/src/main/java/app/lockbook/ui/DrawingView.kt

```diff
@@ -567,6 +586,32 @@ class DrawingView(context: Context, attributeSet: AttributeSet?) :
             } finally {
                 holder.unlockCanvasAndPost(canvas)
             }
+
+            frameTime = (System.nanoTime() - frameStartTime) / 1000000L
+
+            if (frameTime < MIN_TIME_PER_FRAME) {
+                try {
+                    Thread.sleep(MIN_TIME_PER_FRAME - frameTime)
```

Comment:
Thread.sleep is a code smell could you explain what you’re trying to do here?

Reply:
My bad, I forgot to mention that I limited the frames per second to 60 rather than letting it process as many frames as it wants. I did this just cause most devices wouldn't have any use for all the frames it makes, but this can easily be changed so I can max it out at 120 or something. I can also make a setting for this if you want me to and have it so the user can adjust it.